mlflow/pyfunc/__init__.py

```diff
@@ -1054,6 +1098,16 @@ def spark_udf(spark, model_uri, result_type="double", env_manager=_EnvManager.LO
 
     model_metadata = Model.load(os.path.join(local_model_path, MLMODEL_FILE_NAME))
 
+    if params:
```

Comment:
since this mirrors line 424:432, this can probably be a validation function that returns the validated params